apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -16,7 +16,7 @@ const {
 } = require('../constants')
 
 // TODO(pahaz): move if after Organization refactoring
-const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+const { ORGANIZATION_OWNED_FIELD } = require('@app/condo/schema/_common')
```

Comment:
we need to move the ORGANIZATION_OWNED_FIELD to the organization domain `schema/fields.js`